Description: Trigger the use of Debian specific code in PlayOnLinux
 Upstream included most of our patches but inactivated them. To disable update
 alerts, bug reports and statistics, change the locale directory, use the
 msttcorefonts from the Debian package and use the default terminal emulator,
 we just need to set DEBIAN_PACKAGE to TRUE.
Forwarded: not-needed
Author: Bertrand Marc <beberking@gmail.com>
--- a/python/lib/Variables.py
+++ b/python/lib/Variables.py
@@ -26,7 +26,7 @@
 # Debian packagers should switch this to TRUE
 # It will disable update alerts, bug reports, statistics
 # It will set the good locale directory, and it will use the good msttcorefonts
-os.environ["DEBIAN_PACKAGE"] = "FALSE"
+os.environ["DEBIAN_PACKAGE"] = "TRUE"
 
 # Variables PlayOnMac
 if (os.environ["POL_OS"] == "Mac"):
